cabal.project

```diff
@@ -6,3 +6,10 @@ packages: isabelle
 write-ghc-environment-files: never
 tests: true
 test-show-details: streaming
+
```

Comment:
tasty does not get exceptions from quickcheck. This has been fixed in a fork of tasty for now.
https://github.com/yveshauser/tasty/commit/532cd953571e1dd1192bdfece6ee1143ebe488e3

Reply:
We should use a custom tasty ingredient instead to solve this issue and provide better cli options